Makefile

```diff
@@ -2,6 +2,8 @@ BUILDER ?= virtualbox-iso.virtualbox
 VERSION ?= 22.05
 ARCH ?= x86_64
 REPO ?= nixos/nixos
+BUILD_PROVIBE = $(word 2, $(subst ., ,${BUILDER}))
```

Comment:
I don't know the term "provibe". Or did you mean "provider"?

Looks good otherwise